src/videojs-contrib-hls.js

```diff
@@ -278,8 +279,42 @@ export default class HlsHandler extends Component {
       let seekable;
 
       if (!updatedPlaylist) {
+        let selectedPlaylist = this.selectPlaylist();
+
+        if (!selectedPlaylist.attributes.AUDIO) {
+          // from safari defaults
+          this.tech_.audioTracks().addTrack(new AudioTrack({
+            enabled: true,
+            id: '1',
+            kind: 'main',
+            tech: this.tech_
```

Comment:
Yeah I don't think that AudioTracks or VideoTracks should actually require tech_